judges/quality-of-service/quality-of-service.rb

```diff
@@ -105,3 +105,21 @@ def lifetime(type)
   end
 end
 f.average_backlog_size = issues.empty? ? 0 : issues.inject(&:+) / issues.size
+
+# Rejection PR rate
+all_prs_count = 0
+unmerged_prs_count = 0
+Fbe.unmask_repos.each do |repo|
+  all_q = "repo:#{repo} type:pr closed:>#{$SINCE.utc.iso8601[0..10]}"
+  all_prs_count += Fbe.octo.search_issues(all_q)[:total_count]
+
+  unmerged_q = "repo:#{repo} type:pr is:unmerged closed:>#{$SINCE.utc.iso8601[0..10]}"
```

Comment:
@Yegorov same here, this variable is a noise: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html

Reply:
@yegor256 Does it really look better?
```ruby
unmerged_prs_count += Fbe.octo.search_issues("repo:#{repo} type:pr is:unmerged closed:>#{$SINCE.utc.iso8601[0..10]}")[:total_count]
```
what is it
```ruby
unmerged_q = "repo:#{repo} type:pr is:unmerged closed:>#{$SINCE.utc.iso8601[0..10]}"
unmerged_prs_count += Fbe.octo.search_issues(unmerged_q)[:total_count]
```
And in the end I get this
```
judges/quality-of-service/quality-of-service.rb:114:121: C: Layout/LineLength: Line is too long. [133/120]
  unmerged_prs_count += Fbe.octo.search_issues("repo:#{repo} type:pr is:unmerged closed:>#{$SINCE.utc.iso8601[0..10]}")[:total_count]
```
```ruby
unmerged_prs_count += Fbe.octo.search_issues(
  "repo:#{repo} type:pr is:unmerged closed:>#{$SINCE.utc.iso8601[0..10]}"
)[:total_count]
```